apps/src/dropletUtils.js

```diff
@@ -404,7 +417,15 @@ exports.generateAceApiCompleter = function (functionFilter, dropletConfig) {
         callback(null, []);
         return;
       }
-      callback(null, apis);
+      var line = session.getLine(pos.row);
+      var identifier = acUtil.retrievePrecedingIdentifier(line, pos.column);
+      if (pos.column > (identifier.length + 1) &&
+          line[pos.column - identifier.length - 1] === '.') {
```

Comment:
After studying this a little further (check out the strange things regarding `2.toString()` in this article: http://bonsaiden.github.io/JavaScript-Garden/), I've decided that I'm not going to care if there is anything before the `.` character. Which also removes the need for the strange `+ 1` the could be construed as an off-by-one error.